app/graphql/schema/author.js

```diff
@@ -0,0 +1,63 @@
+import { gql } from 'apollo-server-express';
+
+const author = gql`
+    type Author {
+        _id: ID
+        firstName: String!
+        lastName: String!
+        email: String!
```

Comment:
you can look at ```graphql-scalars``` for email type. Check [here](https://www.graphql-scalars.dev/docs/introduction)